### ISSUE TITLE
--force-install addition to compile.sh isn't supported by vmware-install.pl

### ISSUE THREAD

**Issue by mikeputnam** [rasa/vmware-tools-patches#86]
Commit 3c8946b95cec12c0748722bc18796af9618030b4 introduced a `--force-install` parameter to the compile.sh which vmware-install.pl does not appear to support, causing it to emit a `Usage:` block indicating an error. The vmware-install.pl run fails.

```
...
 60400K .......... .......... .......... .......... .......... 99% 1.64M 0s
 60450K .......... .......... .......... .......... .......... 99% 1.64M 0s
 60500K .......... .......... .......... .......... .......... 99% 2.49M 0s
 60550K .......... .......... .......... ..........           100% 1.74M=35s

2015-12-22 16:43:17 (1.67 MB/s) - ‘com.vmware.fusion.tools.linux.zip-7.1.1.tar’ saved [62044160/62044160]

mount: block device /tmp/vmware-tools-patches/downloads/payload/linux.iso is write-protected, mounting read-only
initctl: Job failed while stopping
initctl: Unknown instance:
VMware Tools 9.9.2 build-2496486 for Linux installer
Usage: ./vmware-install.pl [[-][-]d[efault]]
default: Automatically answer questions with the proposed answer.
[[-][-]prefix=<path to install product: bin, lib, doc>]    Put the installation
at <path> instead of the default location.  This implies '--default'.
--clobber-kernel-modules=<module1,module2,...>    Forcefully removes any VMware
related modules installed by any other installer and installs the modules
provided by this installer.  This is a comma seperated list of modules.
...
```

**Comment by mafrosis** [comment]
Sorry to hear you're having problems @mikeputnam! As I wrote in #84, I tested this with multiple versions and everything worked okay.

I just ran a build of Debian targeting `7.1.1` and it was successful. What OS are you building? Are you targeting `7.1.1`? (Which should produce tools `VMwareTools-9.9.2-2496486.tar.gz`).

**Comment by sykesm** [comment]
@mafrosis I just hit this problem in a packer build of [bosh-lite]. I was using `VMware Tools 9.9.4 build-3193940` as shipped with VMware Fusion 7.1.3 with Ubuntu 14.04.3.

If you crack open the [7.1.3 package] that `download-tools.sh` points to, you'll see that the `vmware-install.pl` script on the disk image doesn't have any code in support of `--force-install`.

[bosh-lite]: https://github.com/cloudfoundry/bosh-lite
[7.1.3 package]: https://github.com/rasa/vmware-tools-patches/blob/master/download-tools.sh#L14

**Comment by dragon788** [comment]
It might be advisable to run a test against the script with --force-install to see if it causes a non-zero exit code when trying to access it via help before making it part of the final command.
